eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/rpc/BasicBitcoinJsonRPCClient.scala

```diff
@@ -16,22 +16,24 @@
 
 package fr.acinq.eclair.blockchain.bitcoind.rpc
 
-import com.softwaremill.sttp._
-import com.softwaremill.sttp.json4s._
 import fr.acinq.bitcoin.ByteVector32
 import fr.acinq.eclair.KamonExt
 import fr.acinq.eclair.blockchain.Monitoring.{Metrics, Tags}
 import org.json4s.JsonAST.{JString, JValue}
 import org.json4s.jackson.Serialization
 import org.json4s.{CustomSerializer, DefaultFormats}
+import sttp.capabilities
+import sttp.client3._
+import sttp.client3.json4s._
+import sttp.model.StatusCode
 
 import java.nio.charset.StandardCharsets
 import java.nio.file.{Files, Path}
 import java.util.concurrent.atomic.AtomicReference
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.{Failure, Success, Try}
 
-class BasicBitcoinJsonRPCClient(rpcAuthMethod: BitcoinJsonRPCAuthMethod, host: String = "127.0.0.1", port: Int = 8332, ssl: Boolean = false, wallet: Option[String] = None)(implicit http: SttpBackend[Future, Nothing]) extends BitcoinJsonRPCClient {
+class BasicBitcoinJsonRPCClient(rpcAuthMethod: BitcoinJsonRPCAuthMethod, host: String = "127.0.0.1", port: Int = 8332, ssl: Boolean = false, wallet: Option[String] = None)(implicit sb: SttpBackend[Future, capabilities.WebSockets]) extends BitcoinJsonRPCClient {
```

Comment:
Can you explain why we choose the `WebSockets` capabilities? I don't see what we should choose, we're not actually using websockets but that `capabilites` object doesn't contain anything obvious...

Reply:
Done with df2adc950.